.golangci.yml

```diff
@@ -54,10 +54,6 @@ linters:
 
     # Excluding configuration per-path, per-linter, per-text and per-source
     rules:
-      - linters:
-          - unused
-        path: pagefile.go # This exclusion is required for Windows only
```

Comment:
This exclusion was brought from contrib, irrelevant for this repo. It's also giving a `WARNING` message for every directory causing noise.